src/app/rpm-ostree-countme.sysusers

```diff
@@ -0,0 +1 @@
+u rpm-ostree-countme - "rpm-ostree Count Me"
```

Comment:
Hmm wait why did we stop using `DynamicUser`?

Reply:
I was under the impression that a fixed user was though to be more future proof when we discussed that by I might have misunderstood.

Reply:
Let's re-debate this?  

One thing I feel somewhat strongly about is we should use an unprivileged user for more things in the future, among them fetching repodata and depsolving.  (And probably the same for ostree HTTP fetches, but properly privilege separating that is a lot harder)

Which perhaps argues for a `rpm-ostree-unpriv` user?  Or maybe for short just `rpmostree`?